### PR TITLE
[DESK] Update Turkish (tr-TR) translation

### PR DESCRIPTION
**Pull request by scropool**
## Purpose

I would like to merge the incomplete PR #7308 with the completed version, this time the translations were done more smoothly so im mentioning that to incomplete PR

### CODE REVIEW

**Comment by binarymaster** [comment]
You could have reopened the old one, restore the branch and push to it.

**Comment by scropool** [comment]
> You could have reopened the old one, restore the branch and push to it.

next time I will do so, now let's continue from this PR, unfortunately there are too many deficiencies in the turkish translate files

**Comment by binarymaster** [comment]
Cc for review: @reactos/lang-turkish @eersoy93 @Zaryob @vsaucing @mahirgul @HydraDragonAntivirus

File: dll/cpl/desk/lang/tr-TR.rc [hunk 1]
```diff
@@ -1,4 +1,10 @@
-/* TRANSLATOR: 2013-2020 Erdem Ersoy (eersoy93) <erdemersoy@erdemersoy.net> */
+/*
+ * PROJECT:     ReactOS Display Control Panel
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Turkish resource file
+ * TRANSLATORS: Copyright 2013-2020 Erdem Ersoy (eersoy93) <erdemersoy@erdemersoy.net>
+ *              Copyright 2024 Ethem De Santa (scropool) <oyunyumru4@gmail.com>
+ */
```
Comment (binarymaster) [inline dll/cpl/desk/lang/tr-TR.rc:7]:
```suggestion
/*
 * PROJECT:     ReactOS Display Control Panel
 * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
 * PURPOSE:     Turkish resource file
 * TRANSLATORS: Copyright 2013-2020 Erdem Ersoy (eersoy93) <erdemersoy@erdemersoy.net>
 *              Copyright 2021-2023 Süleyman Poyraz <zaryob.dev@gmail.com>
 *              Copyright 2024 Ethem De Santa (scropool) <oyunyumru4@gmail.com>
 */
```

**Pull request closed without merging**